components/automate-ui/src/app/helpers/auth/regex.ts

```diff
@@ -0,0 +1,12 @@
+export class Regex {
```

Comment:
PR was getting big enough, but probably should rename this class to `Constants` so in addition to `patterns` could also have:
- `messages`, for things like "Only lowercase letters, numbers, and hyphens(-) are allowed." and "must be 64 characters or less"
- `numbers`, for things like the `64` that is used multiple times in `Validators.maxLength(64)`